File: launchdarkly/typescript-node/src/GetMauUsageExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.AccountUsageBetaApi();
apiCaller.setApiKey(api.AccountUsageBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getMauUsage().then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling AccountUsageBeta#getMauUsage:");
  console.log(error.body);
});

File: launchdarkly/ruby/src/GetMauUsageExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::AccountUsageBetaApi.new.get_mau_usage

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AccountUsageBeta#get_mau_usage: #{e}"
end
